src/Sarif.WorkItems/SarifWorkItemModelTransformer.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.WorkItems;
+
+namespace Microsoft.CodeAnalysis.Sarif.WorkItems
+{
+    public abstract class SarifWorkItemModelTransformer
+    {
+        public abstract void Transform(SarifWorkItemModel workItemModel);
```

Comment:
>Transform [](start = 29, length = 9)

Would prefer this to be async if possible. Let's say in my transform method, I'm calling into an async method to retrieve the area path, to override this, it would mean I have to block or do a .Result.

Reply:
Yes, good point.

---
In reply to: [381443233](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381443233) [](ancestors = 381443233)